src/sentry/integrations/github/client.py

```diff
@@ -125,10 +125,9 @@ def get_trees_for_org(
                 try:
                     full_name: str = repo_info["full_name"]
                     branch = repo_info["default_branch"]
-                    trees[full_name] = {
-                        "default_branch": branch,
-                        "files": self.get_tree(full_name, branch),
-                    }
+                    files = self.get_tree(full_name, branch)
+                    repo = Repo(full_name, branch)
+                    trees[full_name] = RepoTree(repo, files)
```

Comment:
Let's see if this will prevent the issue of "files" not being defined for the dictionary.